app/pages/profile/index.cjsx

```diff
@@ -55,45 +63,69 @@ UserProfilePage = React.createClass
       .then ([profileHeader]) =>
         @setState({profileHeader})
 
+  renderNavLinks: ->
+    baseLink = if @props.project? then "/projects/#{@props.project.slug}/" else "/"
+    classes = classNames {
+      "about-tabs": @props.project?
+    }
+    <span>
+      <IndexLink to="#{baseLink}users/#{@props.profileUser.login}" className={classes} activeClassName="active" onClick={logClick?.bind(this, 'comments')}>
+         <Translate content="profile.nav.comments" />
+      </IndexLink>
+      {' '}
+      <Link to="#{baseLink}users/#{@props.profileUser.login}/collections" className={classes} activeClassName="active" onClick={logClick?.bind(this, 'collections')}>
+        <Translate content="profile.nav.collections" />
+      </Link>
+      {' '}
+      <Link to="#{baseLink}users/#{@props.profileUser.login}/favorites" className={classes} activeClassName="active" onClick={logClick?.bind(this, 'favorites')}>
+        <Translate content="profile.nav.favorites" />
+      </Link>
+      {' '}
+
+      <span>
+        {if @props.user is @props.profileUser
+          <Link to="#{baseLink}users/#{@props.profileUser.login}/stats" className={classes} activeClassName="active" onClick={logClick?.bind(this, 'stats')}>
+            <Translate content="profile.nav.stats" />
+          </Link>
+        else
+          <Link to="#{baseLink}users/#{@props.profileUser.login}/message" className={classes} activeClassName="active"onClick={@logClick.bind null, this}>
```

Comment:
Missing space between `activeClassName="active"onClick={@logClick.bind null, this}`